Pages/Index.cshtml.cs

```diff
@@ -24,75 +27,87 @@ public string GetData(string endpoint)
- //               string jsonstring = WebClient.DownloadString("https://data.cityofchicago.org/resource/xq83-jr8c.json");
-                QuickType.Energy[] energies = QuickType.Energy.FromJson(jsonstring);
-
-                jsonstring = GetData("https://data.cityofchicago.org/resource/tfm3-3j95.json");
-//                jsonstring = WebClient.DownloadString("https://data.cityofchicago.org/resource/tfm3-3j95.json");
-                QuickTypeVehicle.Vehicle[] vehicles = QuickTypeVehicle.Vehicle.FromJson(jsonstring);
-
-                List<Affluence> affluence = new List<Affluence>();
-
-                var energy_query = from energy in energies
-                                  group energy by new { energy.ZipCode, energy.Latitude, energy.Longitude } into g
-                                  select new
-                                  {
-                                      Zip = g.Key.ZipCode,
-                                      Latitude = Math.Round(g.Key.Latitude, 0),
-                                      Longitude = Math.Round(g.Key.Longitude, 0),
-                                      PowerUsage = Math.Round(g.Sum(u => u.ElectricityUseKbtu) / g.Sum(u => u.GrossFloorAreaBuildingsSqFt), 0)
-
-                                      ////FloorSize = energy.GrossFloorAreaBuildingsSqFt,
-                                      //TotalUsage = g.Sum(u=> u.ElectricityUseKbtu),
-
-                                  };
-
-                var vehicle_query = from vehicle in vehicles
-                                    group vehicle by new { vehicle.ZipCode} into k
-                                    select new
-                                    {
-                                        Zip = k.Key.ZipCode,
-                                        vehicleCount = k.Count()
-                            };
-
-                var final_query = from energy in energy_query
-                                  join vehicle in vehicle_query
-                                  on energy.Zip equals vehicle.Zip
-                                  select new
-                                  {
-                                      Zip = energy.Zip,
-                                      Latitude = energy.Latitude,
-                                      Longitude = energy.Longitude,
-                                      PowerUsage = energy.PowerUsage,
-                                      VehicleCount = vehicle.vehicleCount
-                                  };
-
-                List<Affluence> target = new List<Affluence>();
-
-                foreach (var item in final_query)
-                {
 
+                energies = Energy.FromJson(energyjsonstring);
+            }
+            string vehiclejsonstring = GetData("https://data.cityofchicago.org/resource/tfm3-3j95.json");
+            //                jsonstring = WebClient.DownloadString("https://data.cityofchicago.org/resource/tfm3-3j95.json");
```

Comment:
Remove commented out code.  It's considered bad practice to commit commented-out code to version control, because you can always go back in history to see what used to be there.